solutions/swb-app/src/users/updateUserRequest.ts

```diff
@@ -8,6 +8,7 @@ import { z } from 'zod';
 // eslint-disable-next-line @rushstack/typedef-var
 export const UpdateUserRequestParser = z
   .object({
+    userId: z.string().min(1),
     firstName: z.string().optional(),
```

Comment:
do these also need mins?

Reply:
this is a required value and there is no static route config to avoid empties, so adding the regex later may get rid of this

Reply:
this is going to be addressed in the validation input tasks